### PR TITLE
optimize internlm xcomposer2 performance

### PR DESCRIPTION
**Pull request by MeouSker77**
## Description

optimize internlm xcomposer2 performance

### 2. User API changes

use following code to load model to get best performance (`sym_int8` for better accuracy, `sym_int4` for better speed)

```python
from ipex_llm.transformers import AutoModelForCausalLM
model = AutoModelForCausalLM.from_pretrained(model_path, trust_remote_code=True,
                                             load_in_low_bit="sym_int8", modules_to_not_convert=["vit"])
model = model.half()
model = model.eval()
```

a complete example
```python
import time
import torch

from ipex_llm.transformers import AutoModelForCausalLM
from transformers import AutoTokenizer

ckpt_path = "internlm-xcomposer2-vl-7b"
tokenizer = AutoTokenizer.from_pretrained(ckpt_path, trust_remote_code=True)
model = AutoModelForCausalLM.from_pretrained(ckpt_path, trust_remote_code=True,
                                             load_in_low_bit="sym_int8", modules_to_not_convert=["vit"])
model = model.half()
model = model.eval()

print(model)

query = '<ImageHere>Please describe this image in detail.'
image = 'image1.webp'

model = model.to('xpu')

with torch.inference_mode():
    for i in range(3):
        st = time.time()
        response, _ = model.chat(tokenizer, query=query, image=image, history=[], do_sample=False, max_new_tokens=1)
        et = time.time()
        print(response)
        print(et - st)
```

### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

**Comment by MeouSker77** [comment]
PR validation: <https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/9869875195>

**Review by rnwang04 — APPROVED** [review]
